docs/index.asciidoc

```diff
@@ -1,48 +1,9 @@
 [[kibana-guide]]
 = Kibana User Guide
 
-//////////
-release-state can be: released | prerelease | unreleased
-//////////
-:release-state:  unreleased
-:version:        5.5.0
-:major-version:  5.x
-:branch:         5.x
+/////
+repos can be open | all
+/////
+:repos:             open
```

Comment:
I can't tell for sure without seeing index-shared, but perhaps this variable name could be a bit more descriptive?  Maybe something like `repo-state`?